### PR TITLE
Add eslint test to travisci

### PR DESCRIPTION
**Pull request by Dean-Coakley**
Fixes: https://github.com/davej/CorkJS/issues/15
* Not sure if _James Morrissey_ wishes to be credited or not
* Not sure if all these fields in `package.json` are required, maybe some should be removed. E.g license
* Should `.eslintrc.js` be in root of the repo as it is?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "corkjs",
+  "version": "1.0.0",
+  "description": "Displays information about CorkJs powered by meetup.com api",
+  "main": ".eslintrc.js",
```
Comment (davej) [inline package.json]: Should probably remove the `main` field altogether. I don't think it's required anyway.

**Comment by davej** [comment]
Good job 👍 

Maybe just remove the `main` field from `package.json` and add `.DS_Store` to the `.gitignore` file.

**Comment by davej** [comment]
Oh looks like there is no `.gitignore` file. Perhaps create one and add `node_modules/` to it too.

**Comment by Dean-Coakley** [comment]
@davej Should travisci be showing the two warnings in github?

**Comment by davej** [comment]
No, travis only fails on errors, not warnings. You can make them errors if you want them to cause a failing build.

**Comment by davej** [comment]
Feel free to merge this if you're happy 🙂

**Comment by davej** [comment]
Probably makes sense to do a squash merge though.

**Comment by Dean-Coakley** [comment]
@davej updateDOM is giving unused-var warning/error. https://github.com/davej/CorkJS/blob/master/scripts/main.js#L19 But I am referencing in the callback of the url.
Any ideas on how to fix this? The only solutions I can think of require ES6

**Comment by Dean-Coakley** [comment]
Of course, it's not necessary. Just curious

**Comment by davej** [comment]
Two options:

1. Explicitly opt-out of the rule with a comment (syntax of comment might be wrong, google it if it doesn't work).

```js
// eslint-disable-next-line no-unused-vars
var updateDOM = function (responseJSON) {
```

2. Put the function on the global scope (`window`).

```js
window.updateDOM = function (responseJSON) {
```

I think I prefer option 2 because the function is implicitly on the global scope anyway (because we're not using an [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression)) and it needs to be (because the JSONP callback needs to access it). So it's best to be explicit about it.

**Comment by Dean-Coakley** [comment]
oops I meant to squash my commits, ah well

**Comment by davej** [comment]
Thanks 👍

**Pull request merged**